templates/data/kyma-integration-gardener-data.yaml

```diff
@@ -42,6 +42,20 @@ templates:
               pipeline.test: "fast-integration"
               pipeline.clusterprovisioning: kyma cli
             command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/kyma-integration-gardener-eventing.sh"
+          command_integration_skr_eventing:
+            cluster: "trusted-workload"
```

Comment:
Does this job need to run on trusted cluster?

Reply:
updated